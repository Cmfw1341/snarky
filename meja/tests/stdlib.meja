let test_plus = 1i + 2i;

let test_times = 3i * 4i;

let test_print_int = print_int(15i);

let fold_list = fun (l : list(bool)) => {
  List.fold_left ((fun (x, y) => {x && y;}), true, l);
};
