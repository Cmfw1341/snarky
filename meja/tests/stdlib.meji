let test_plus : int;

let test_times : int;

let test_print_int : unit;

let fold_list : list(bool) -> bool;


