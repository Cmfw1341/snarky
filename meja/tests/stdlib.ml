open Snarky
open Snarky.Snark
module Impl =
  Snarky.Snark.Run.Make (Snarky.Backends.Mnt4.Default) (Core_kernel.Unit)
open Impl

let test_plus = 1 + 2

let test_times = 3 * 4

let test_print_int = print_int 15

let fold_list (l : bool list) = List.fold_left (fun x y -> x && y) true l
