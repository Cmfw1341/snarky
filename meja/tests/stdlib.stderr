File "tests/stdlib.meja", line 8, characters 45-49:
Error: Unbound constructor true.
